### PR TITLE
Ticket announcements on website!

### PR DESCRIPTION
**Pull request by libbyberrie**
This is mostly last year's "Attend" page, but '24-ified. 

this is technically a WIP at time of PR (5 sep) - could hypothetically be published as is but some polish wouldn't go amiss. 

NICE-TO-HAVE TODO: 

- [x] Update vols page - close vols applications
- [ ] ~~Also publish getting there page~~ other pr
- [ ] ~~(libby's being a pedant) Check accessibility of duration text on program page, particularly on Yellow and Red~~ this is done in pretix

### CODE REVIEW

File: src/components/PurchaseButton.astro [hunk 1]
```diff
@@ -17,15 +17,17 @@ import Button from "./Button.astro"
...
-            li: "https://pretix.eu/pyconau/2023/c/QsEV6PAEj/",
+            li: "https://pretix.eu/pyconau/2024/c/Z4m7ctrPC/",
             // Fediverse
-            fv: "https://pretix.eu/pyconau/2023/c/zEotuIaUC/",
+            fv: "https://pretix.eu/pyconau/2024/c/1mPdQLtHo/",
+
+            // Uncomment and update these values if we end up paying for ads. 
             // Twitter Ads
-            ta: "https://pretix.eu/pyconau/2023/c/PXCb4Blaj/",
+            // ta: "https://pretix.eu/pyconau/2023/c/PXCb4Blaj/",
```
Comment (elena) [inline src/components/PurchaseButton.astro:28]: I'd argue that these could be deleted, maybe can leave a comment if we really want to remember this, but these are pretty unlikely and links obviously old.

File: src/content/pages/attend/_index.md [hunk 2]
```diff
@@ -1,10 +1,9 @@
...
 title: Attend PyCon AU
-order: 1
 ---
 
 # More details coming soon!
 
 We'll have more information about how you can attend PyCon AU coming soon.
 
-In the meantime, If you would like to apply for a volunteering position at this year's conference, please [visit the volunteers page](/attend/volunteer) for more information.
+In the meantime, If you would like to apply for a volunteering position at this year's conference, please [visit the volunteers page](/attend/volunteer) for more information.
```
Comment (elena) [inline src/content/pages/attend/_index.md:9]: In the meantime, **I**f you would <-- probably lower case `I` in `if`

**Comment by elena** [comment]
SQUEE! tyty :sparkling_heart:

File: src/content/pages/attend/_index.md (hunk 2 above)
Comment (libbyberrie) [reply src/content/pages/attend/_index.md:9]: I've made that change, but the page has been hidden from view anyway (the thing i changed was adding a `_` to the filename, effectively unpublishing it.

File: src/content/pages/attend/_volunteer-closed.mdx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+### Become a session chair closer to the event!
+
+Session chairs are an integral part of the conference experience, as they are responsible for introducing and sending off each speaker during the main conference. 
+We will send more information to ticket holders about becoming a session chair in an email to the address you provided, within the weeks coming up to the event. 
+
+### Know someone who would make a good sponsor? Contact us!
+
+We're still looking for companies who want to support this event! We especially want to get in contact with Australian small businesses and community sponsors who want to give back to the local python community. If that sounds like you, or you know of a company that would be interested, email us at [sponsor@pycon.org.au](mailto:sponsor@pycon.org.au).
+See our [sponsors page](/sponsor/) for more details.
```
Comment (libbyberrie) [inline src/content/pages/attend/_volunteer-closed.mdx:27]: @devjack let me know if there's anything different you would like to place here.

Comment (libbyberrie) [reply src/content/pages/attend/_volunteer-closed.mdx:27]: (Jack confirmed over discord that it looked good.)

File: src/content/pages/attend/volunteer.mdx [hunk 4]
```diff
@@ -55,7 +64,7 @@ PyCon AU operates under a clear [Code of Conduct](/safety) and we ask that all v
 
 If you are interested, please use the form below to send us your details, the times that you are available, and the skills you have to offer.
 
-Volunteer applications close midnight Melbourne time at the end of **Monday 07 October, 2024**.
+Volunteer applications close 5PM AEST on **Wednesday 18 September, 2024**.
```
Comment (urcheraus) [inline src/content/pages/attend/volunteer.mdx:67]: Might be nice to keep the old time here with ~~strikethrough~~ for transparency and put in something like "Update: due to the record number of volunteer applications we are closing volunteer applications earlier than originally planned"

**Pull request merged**